docs/webiny-apps/page-builder/development/creating-iframe-element-plugin.md

```diff
@@ -31,29 +31,36 @@ Head over to our [Webiny forum](https://community.webiny.com/), [Gitter](https:/
 
 ### Editor
 
-One of the main sections of the Page Builder App is the Editor, you can get there by editing a site that you create. By default, it offers you a variety of elements or plugins as seen in the image below.
+One of the main sections of the Page Builder App is the Editor, you can open the Editor by editing the page that you create.
+Out-of-the-box the PB editor includes a myriad of page elements.
 
 ![Editor Elements](/img/webiny-apps/page-builder/development/development/plugin-reference/editor/iframe/editor-elements.png)
 
 
-When building pages, you are using these elements (e.g. Text, Image, Form, ...), which are registered in the system via plugins. In terms of rendering, every element has to have two plugins, one that defines how it's rendered in the actual page builder, and the other that defines how it's rendered on the website.
+This list of page elements can be expanded and custom page elements can be created via plugins. In order to create a new page element, we need to register two plugins, one that defines how it's rendered in the actual page builder, and the other that defines how it's rendered on the website.
 
 If you are interested on different types of plugins, each app in Webiny consistes of the `types.ts` that includes all the plugin types of that app.
```

Comment:
> If you are interested on different types of plugins, each app in Webiny consistes of the types.ts that includes all the plugin types of that app. In our case is app-page-builder/src/types.ts

This is the section I thought we'd remove, and instead of linking the source code on GH, we'd link the newly created entries in the plugin reference.

But, since you'll be mentioning the needed plugins below and you already linked them, I think we can remove this section completely.